Samples/Desktop/D3D12Raytracing/src/D3D12RaytracingBasicShaderExecutionReordering/Raytracing.hlsl

```diff
@@ -160,10 +160,10 @@ void MyRaygenShader()
     RenderTarget[DispatchRaysIndex().xy] = color;
 }
 
-    
+
 [shader("closesthit")]
-void MyClosestHitShader(inout RayPayload payload, in MyAttributes attr)
-{
+void ClosestHit_Complex(inout RayPayload payload, in MyAttributes attr)
+    {
```

Comment:
Are the complex and simple hit shaders different in any way?  I wouldn't have expected each of them to have complex vs simple branches in them.  And would have expected at least some more complexity in the one labelled as "complex".